### DISCUSSION TITLE
Technitium Update Error

### DISCUSSION THREAD

**Discussion by DJKatastrof** [tteck/Proxmox#2435] [Help Needed] [answered]
Hey, trying to update Technitium but getting these errors,
Any ideas? :)

![image](https://github.com/tteck/Proxmox/assets/64254359/b45dc37f-4d5e-4409-bafb-bb70a38dce8b)

**Comment by NitroHype** [comment]
I tried this also and have the same message. I typed "update" and tried the full command line.

**Comment by Pid226** [comment]
Good thing I thought to double check here before applying the update and saw your post.  Looking at the change notes for Version 12.0 https://github.com/TechnitiumSoftware/DnsServer/blob/master/CHANGELOG.md.  They're using .NET 8 now .  Reading the install script, it's instaling .NET 7 so I'm guessing thats the cause for the error.

I just ran through the install script on a dev box changing "$STD apt-get install -y aspnetcore-runtime-7.0" to "$STD apt-get install -y aspnetcore-runtime-8.0".  It seems to have completed fine and I'm not seeing any errors so far.  

Just in case, I'm going to hold fire doing production till the script is updated.

**Comment by tteck** [comment, marked as answer]
Fixed
